pkgs/by-name/ne/netgen/package.nix

```
@@ -0,0 +1,192 @@
+    ./conditional-arm-feature-complex.patch
+
+    # when generating python stub file utilizing system python pybind11_stubgen module
+    # cmake need to inherit pythonpath
+    ./inherit-env-pythonpath.patch
+
+    ./find-git-not-mendatory.patch
+    (fetchDebianPatch {
+      inherit (finalAttrs) pname;
+      version = "6.2.2404+dfsg1-4";
+      patch = "fix-national-encoding.patch";
+      hash = "sha256-oo29H/SN+c/yojtEkFUG99Gc+hJd5sNxZfxV5TzPtRY=";
+    })
+    (fetchDebianPatch {
+      inherit (finalAttrs) pname;
+      version = "6.2.2404+dfsg1-4";
+      patch = "fix_nggui_tcl.patch";
+      hash = "sha256-ODDT67+RWBzPhhq/equWsu78x9L/Yrs3U8VQ1Uu0zZw=";
+    })
+    (fetchDebianPatch {
+      inherit (finalAttrs) pname;
+      version = "6.2.2404+dfsg1-4";
+      patch = "ffmpeg_link_libraries.patch";
+      hash = "sha256-S02OPH9hbJjOnBm6JMh6uM5XptcubV24vdyEF0FusoM=";
+    })
+    (fetchDebianPatch {
+      inherit (finalAttrs) pname;
+      version = "6.2.2404+dfsg1-4";
+      patch = "size_t_int32.patch";
+      hash = "sha256-uudf3b97J2TNq4lAzPK1bRrcQv+Z1oVxFE3tKlydJfE=";
+    })
+    (fetchDebianPatch {
+      inherit (finalAttrs) pname;
+      version = "6.2.2404+dfsg1-4";
+      patch = "link_atomic.patch";
+      hash = "sha256-Yf0GNP4BAahWxOO0zIMVvXiROVP0hRM54Fok7jGABlY=";
+    })
+    (fetchDebianPatch {
+      inherit (finalAttrs) pname;
+      version = "6.2.2404+dfsg1-4";
+      patch = "include_stdlib.patch";
+      hash = "sha256-W+NgGBuy/UmzVbPTSqR8FRUlyN/9dl9l9e9rxKklmIc=";
+    })
+    (fetchDebianPatch {
+      inherit (finalAttrs) pname;
+      version = "6.2.2404+dfsg1-4";
+      patch = "ngappinit_no_MPI.patch";
+      hash = "sha256-0PSe5YB0C/goYFvnl9Z+pwM12D4s1qaTRV12/NA7c94=";
+    })
+  ];
+
+  postPatch = ''
+    # get version via version.txt rather than git describe
+    echo "v${finalAttrs.version}-0" > version.txt
+    # create dummy catch2 target and use system catch2
+    echo "add_custom_target(project_catch)" > cmake/external_projects/catch.cmake
+  '';
+
+  nativeBuildInputs = [
+    cmake
+    makeWrapper
+    python3Packages.pybind11-stubgen
+  ];
+
+  buildInputs = [
+    metis
+    opencascade-occt
+    zlib
+    tcl
+    tk
+    libGLU
+    xorg.libXmu
+    libjpeg
+    ffmpeg
+    mpi
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    packaging
+    pybind11
+    mpi4py
+    numpy
+  ];
+
+  cmakeFlags = [
+    (lib.cmakeFeature "CMAKE_CXX_FLAGS" archFlags)
+    (lib.cmakeBool "USE_MPI" true)
+    (lib.cmakeBool "USE_MPI4PY" true)
+    (lib.cmakeBool "PREFER_SYSTEM_PYBIND11" true)
+    (lib.cmakeBool "BUILD_STUB_FILES" true)
+    (lib.cmakeBool "USE_SUPERBUILD" false) # use system packages
+    (lib.cmakeBool "USE_NATIVE_ARCH" false)
+    (lib.cmakeBool "USE_JPEG" true)
+    (lib.cmakeBool "USE_MPEG" true)
+    (lib.cmakeBool "BUILD_TESTING" finalAttrs.finalPackage.doInstallCheck)
+    (lib.cmakeBool "ENABLE_UNIT_TESTS" finalAttrs.finalPackage.doInstallCheck)
```

```suggestion
    (lib.cmakeBool "BUILD_TESTING" finalAttrs.finalPackage.doCheck)
    (lib.cmakeBool "ENABLE_UNIT_TESTS" finalAttrs.finalPackage.doCheck)
```

Unit tests are not `installCheck`?